src/kv/commonKeyValueDao.ts

```diff
@@ -53,7 +53,7 @@ export type CommonKeyValueDaoSaveOptions = CommonKeyValueDBSaveBatchOptions
 // todo: logging
 // todo: readonly
 
-export class CommonKeyValueDao<T> {
+export class CommonKeyValueDao<T, Key extends string = string> {
```

Comment:
Ok, after thinking for a bit - I agree. It can be part of our future strategy to "support branded string ids in all tools".

I'll just ask you to rename the 2 generics. I think generics normally are typed in UPPERCASE. I propose this naming (one of the variants):

```
CommonKeyValueDao<V, K>
CommonKeyValueDao<VALUE, KEY>
```

I like default `T`, but when the number of generics start to be more than one - I believe it needs clarification and cannot stay just T, hence my suggestion above.